queries/ne.jinja2

```diff
@@ -247,7 +246,7 @@ SELECT
   NULL::jsonb AS __places_properties__,
 
   jsonb_build_object(
-      'min_zoom', GREATEST(scalerank, 5),
+      'min_zoom', scalerank,
       'scalerank', scalerank,
```

Comment:
This instance in particular was replaced with https://github.com/tilezen/vector-datasource/blob/6b150e2bae1e4d322c6fdf50b978a34aa45a33e1/yaml/roads.yaml#L123